Subject: [PATCH] m68k: Ratelimit ide_release_lock bug messages

Ratelimit the annoying ide_release_lock bug messages as attempting to release
the already released lock does not appear to cause any harm in practice.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 include/asm-m68k/ide.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/include/asm-m68k/ide.h
+++ b/include/asm-m68k/ide.h
@@ -114,7 +114,8 @@ static __inline__ void ide_release_lock 
 {
 	if (MACH_IS_ATARI) {
 		if (falconide_intr_lock == 0) {
-			printk("ide_release_lock: bug\n");
+			if (printk_ratelimit())
+				printk("ide_release_lock: bug\n");
 			return;
 		}
 		falconide_intr_lock = 0;
